run-tests.php

```diff
@@ -1688,9 +1698,9 @@ function kill_children(array $children)
     }
 }
 
-function run_worker()
+function run_worker($workerID)
```

Comment:
nit: It seems inconsistent to have this as a global variable in some places but a regular parameter with the same value here

Reply:
In an effort to move away from the global problem, this is the beginning of
many PRs.

Im trying to keep this PR tiny and focus on HTML bug fix, so i can come
back with another PR with many improvements.

I've started with this small win here, on this line and there will be more
to come.